WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -411,6 +416,36 @@ private extension StoreStatsAndTopPerformersViewController {
         }
     }
 
+    @MainActor
+    func configureCustomRangeTab() async {
+        guard featureFlagService.isFeatureFlagEnabled(.customRangeInMyStoreAnalytics) else {
+            return
+        }
+
+        guard let customRange = await loadCustomRangeTab() else {
+            return
+        }
+
+        createCustomRangeTab(range: customRange)
+    }
+
+    @MainActor
+    func loadCustomRangeTab() async -> StatsTimeRangeV4? {
+        guard featureFlagService.isFeatureFlagEnabled(.customRangeInMyStoreAnalytics) else {
+            return nil
+        }
+
+        return await withCheckedContinuation { continuation in
+            stores.dispatch(AppSettingsAction.loadCustomStatsTimeRange(siteID: siteID) { timeRange in
+                continuation.resume(returning: timeRange)
+            })
+        }
+    }
+
+    func saveCustomRangeTab(timeRange: StatsTimeRangeV4) {
```

Comment:
nit: If we decide to rename to load method we can rename this into `saveTimeRangeForCustomRangeTab(_ timeRange: StatsTimeRangeV4)` for consistency. Please feel free to improve my suggested name.